docs/sop/DEPLOYING_TO_PRODUCTION.md

```diff
@@ -0,0 +1,325 @@
+| bastion.gigadb.org | eip-gigadb-bastion-live-gigadb    |  live |
+| files.gigadb.org | eip-gigadb-files-live-gigadb      |  live |
+| portainer.gigadb.org | eip-gigadb-live-gigadb            |  live |
+| staging.gigadb.org | eip-gigadb-staging-gigadb         |  staging |
+| bastion-stg.gigadb.host | eip-gigadb-bastion-staging-gigadb |  staging |
+| files.staging.gigadb.org | eip-gigadb-files-staging-gigadb   |  staging |
+| portainer.staging.gigadb.org | eip-gigadb-staging-gigadb         |  staging |
+
+
+#### Hot Stand-by
+
+| FQDN | EIP |  environment |
+| --- | --- | --- |
+| alt-live.gigadb.host | eip-gigadb-live-gigadb  |  live |
+| bastion.alt-live.gigadb.host | eip-gigadb-bastion-live-gigadb |  live |
+| files.alt-live.gigadb.org | eip-gigadb-files-live-gigadb |  live |
+| portainer.alt-live.gigadb.host |  eip-gigadb-live-gigadb |  live |
+| alt-staging.gigadb.host | eip-gigadb-staging-gigadb  |  staging |
+| bastion.alt-staging.gigadb.host | eip-gigadb-bastion-staging-gigadb |  staging |
+| files.alt-staging.gigadb.host | eip-gigadb-files-staging-gigadb |  staging |
+| portainer.alt-staging.gigadb.host |  eip-gigadb-staging-gigadb |  staging |
+
+### Pipeline and provisioning
+
+You can now follow the process described in `docs/SETUP_CI_CD_PIPELINE.md` and `docs/SETUP_PROVISIONING.md` 
+while remembering that:
+* for terraform, prefix the command with `AWS_PROFILE=Upstream` or `AWS_PROFILE=UpstreamAlt` depending on which role you are deploying
+* for Ansible, make sure to configure one of the AWS regions in the table at the top depending on which role you are deploying
+
+## Keeping data in sync
+
+The relevant data stores are:
+* EFS for the user dropboxes
+* RDS for the files metadata
+
+Regularly, the data from both stores on the current production needs to be synchronised to the hot stand-by.
+
+| Data store | method | Frequency/time | what infra performs the synching                     | script                                                            |
+| --- | --- | --- |------------------------------------------------------|-------------------------------------------------------------------|
+| EFS | rlcone sync | daily | hot stand-by                                         | sync_drobox                                                       |
+| RDS | pgsql load from RDS backup | daily | current production (backup) + hot stand-by (restore) | s3backup (on current production), databaseReset (on hot stand-by) |
+
+>**Note**: TODO: `s3backup` and `databaseReset` needs to understand backup keyed on date (YYYYMMDD) and date time (YYYYMMDDHHMM). 
+> The former is used for cron driven daily backup and restore, while the latter is for manual backup and restore, during blue/green deployment and other ad-hoc situations.
+
+
+## Deploying releases
+
+
+Both Gitlab projects track `gigasciencve/gigadb-website` repo on GitHub 
+and will deploy to their respective staging production environments automatically.
```

Comment:
Missing `and` in `staging production`?

Reply:
Hi @pli888 
not really.  "production environments" is named as such in opposition to"developer environments".
Like for "developer environments", "production environments" are either staging and live, or with the full naming, of a "staging production environment" and a "live production environment"